student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,49 @@
-import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
-import Navbar from "../Navbar/Navbar"
-import Sidebar from "../Sidebar/Sidebar"
-import Home from "../Home/Home"
-import "./App.css"
+import * as React from "react";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Home from "../Home/Home";
+import "./App.css";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import ProductDetails from "../ProductDetails/ProductDetails";
 
 export default function App() {
+  // const url = "https://codepath-store-api.herokuapp.com/store";
+  const [products, setProducts] = useState([]);
+  const [searchValue, setSearchValue] = useState("");
+  const [searchCatValue, setSearchCatValue] = useState("");
+  const [catValue, setCatValue] = useState("");
+
```

Comment:
recommend lowering all state besides `products` to avoid unnecessary prop drilling